en/snap/integration-guide.md

```diff
@@ -411,55 +437,103 @@ Or try the demo here:
   " class="my-btn">Try Snap Demo &#9099;</button>
 </p>
 
-After payment completed, customer will be redirected back to `Finish URL` [specified on Midtrans Dashboard](/en/snap/advanced-feature.md#configure-redirection-url), under menu **Settings > Snap Preference > System Settings > `Finish URL`**.
+
+After the payment is completed, customer is redirected back to `Finish URL`. It is specified on [Midtrans Dashboard](/en/snap/advanced-feature.md#configure-redirection-url), under menu **Settings > Snap Preference > System Settings > `Finish URL`**.
+
+<details>
+<summary><b>Configuring Finish Redirect URL</b></summary>
+<article>To configure the *Finish Redirect URL*, follow the steps given below.
+
+1. Login to your MAP account.
+
+2. On the Home page, go to **SETTINGS > CONFIGURATION**.
+   *Configuration* page is displayed.   
+
+3. Enter **Finish Redirect URL** with your landing page endpoint.
+
+4. Click **Update**.
+   A confirmation message is displayed.
+
+   ![Core API](/Users/rashmi/Desktop/technical-documentation-site/asset/image/coreapi/core-api-finish-redirect-url-2.png)
+
+   The *Finish Redirect URL* is configured.
+
+   </article>
+   </details>
 
 > **Tips:** Optionally, you can also [use Javascript callbacks](/en/snap/advanced-feature.md#javascript-callback) to handle payment events triggered from customer finishing interaction with Snap payment page.
+>
+> <!--Follow the style for Tips, Notes like ?>***Notes***: or ?>***Tips***:-->
 
-## 3. Create Test Payment
+### 3. Creating Test Payment
 
-Create a test payment to make sure you have integrated Snap successfully. You can use one of our test credentials for Card Payment:
+Create a test payment to make sure you have integrated Snap successfully. There are various payment methods available on Snap. You can choose any one of them to create a test payment. Following are the test credentials for Card payment. 
 
 Name | Value
 --- | ---
 Card Number | `4811 1111 1111 1114`
 CVV | `123`
-Exp Month | Any month (e.g: `02`)
-Exp Year | Any future year (e.g: `2025`)
+Exp Month | Any month in MM format. For example, `02`. 
+Exp Year | Any future year, in YYYY format. For example, `2025`. 
 OTP/3DS | `112233`
 
-You can use more test payment credentials [available on our sandbox payment simulator](/en/technical-reference/sandbox-test.md)
+For more details, refer to [Testing Payments on Sandbox](/en/technical-reference/sandbox-test.md).
```

Comment:
What we are trying to emphasise is: "there are other payment testing credentials for other payment methods in this link", I think "For more details" isn't quite enough explain that